packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -739,18 +752,19 @@ test.serial('bulk operation BULK_GRANULE_REINGEST reingests list of granule IDs
       granule.granuleId === callArgs[0].granule.granuleId);
 
     t.true(t.context.executionArns.includes(callArgs[0].granule.execution));
-
     delete matchingGranule.execution;
     delete callArgs[0].granule.execution;
 
-    t.deepEqual(matchingGranule, callArgs[0].granule);
+    const omitList = ['dataType', 'version'];
+
+    t.deepEqual(omit(matchingGranule, omitList), callArgs[0].granule);
```

Comment:
Sorry for being annoying, but now that I understand better, can we change this to the pattern using `getUniqueGranuleByGranuleId` and `translatePostgresGranuleToApiGranule` for consistency?

Reply:
Not gonna hold up the PR on this though